packages/react-component-library/src/components/Timeline/__tests__/Event.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult, cleanup } from '@testing-library/react'
+
+import { TimelineProvider } from '../context'
+import { Event } from '../Event'
+
+describe('Event', () => {
+  let wrapper: RenderResult
+  let status: string
+  let title: string
+
+  describe('default props', () => {
+    beforeEach(() => {
+      wrapper = render(
+        <TimelineProvider>
+          <Event startDate={new Date()} endDate={new Date()} />
+        </TimelineProvider>
+      )
+    })
+
+    describe('and the status and title props are provided', () => {
+      beforeEach(() => {
+        cleanup()
```

Comment:
It feels like the `cleanup` does not need to be used here as it's testing a completely different scenario so the `and the status and title props are provided` `describe` would not be inside the `default props` `describe`.